packages/@o3r/workspace/package.json

```
@@ -22,7 +22,7 @@
   "main": "./dist/src/public_api.js",
   "types": "./dist/src/public_api.d.ts",
   "peerDependencies": {
-    "@angular-devkit/architect": "^0.1900.0",
+    "@angular-devkit/architect": "~0.1901.0",
```

```suggestion
    "@angular-devkit/architect": "^0.1900.0",
```
peer dependencies should not change right?